src/components/ViewerToolbar.vue

```diff
@@ -63,12 +64,21 @@
         <v-radio value="multiple" label="Multiple" class="smaller" />
         <v-radio value="unroll" label="Unroll" class="smaller" />
       </v-radio-group>
+      
     </div>
     <div class="lowertools">
       <slot></slot>
     </div>
+    
   </div>
 </template>
+
+<style lang="scss" scoped>
+.my-checkbox::v-deep .v-input__control {
+  transform: scale(0.9) translateY(5%);
+}
+</style>
```

Comment:
Instead of creating another block for style, you can use the existing one

Reply:
Thanks!